azure-devops/run-build.yml

```diff
@@ -62,9 +62,8 @@ jobs:
         -host_arch=amd64 -arch=${{ parameters.vsDevCmdArch }} -no_logo
         cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE=$(vcpkgLocation)\scripts\buildsystems\vcpkg.cmake ^
         -DVCPKG_TARGET_TRIPLET=${{ parameters.targetPlatform }}-windows -DCMAKE_CXX_COMPILER=cl ^
-        -DCMAKE_BUILD_TYPE=Release -DLIT_FLAGS=$(litFlags) ^
-        -DCMAKE_CXX_FLAGS=/analyze ^
-        -S $(Build.SourcesDirectory) -B $(buildOutputLocation)
+        -DCMAKE_BUILD_TYPE=Release -DLIT_FLAGS=$(litFlags) -DCMAKE_CXX_FLAGS=/analyze:autolog- ^
+        -DSTL_IS_KERNEL=ON -S $(Build.SourcesDirectory) -B $(buildOutputLocation)

```

Comment:
`-DSTL_IS_KERNEL=ON` is necessary, but not sufficient.  You also need something like the following:
```
    -DWDK_INCLUDE_DIRECTORY="C:/Program Files (x86)/Windows Kits/10/Include/10.0.19041.0"^
    -DWDK_LIB_DIRECTORY="C:/Program Files (x86)/Windows Kits/10/Lib/10.0.19041.0"^
    -DWDK_BIN_DIRECTORY="C:/Program Files (x86)/Windows Kits/10/bin/10.0.19041.0"
```